config/routes.rb

```diff
@@ -28,5 +28,6 @@
   patch '/favorites/:pet_id', to: 'favorites#update'
   delete '/favorites/:pet_id', to: 'favorites#delete'
   get '/applications/new', to: 'adopt_applications#new'
+  get '/applications/:id', to: 'adopt_applications#read'
```

Comment:
good to make sure that new's above id so that it doesn't look for an id of "new"